Goal: Avoid warning breaking the compilation.

Daniel de Rauglaudre:

> ocamlopt.opt -warn-error A -I ../wserver -I ../dag2html -I `camlp4 -where` -c -impl gutil.ppo
> File "gutil.ml", line 1106, characters 6-13:
> Warning Y: unused variable surname.
> Daniel, hints?

Edit tools/Makefile.inc and change:

OCAMLC=ocamlc.opt -warn-error A
OCAMLOPT=ocamlopt.opt -warn-error A

into:

OCAMLC=ocamlc.opt
OCAMLOPT=ocamlopt.opt


Status wrt upstream: Unknown

Index: geneweb-4.10/tools/Makefile.inc.opt
===================================================================
--- geneweb-4.10.orig/tools/Makefile.inc.opt	2005-11-09 07:39:36.643716287 +0100
+++ geneweb-4.10/tools/Makefile.inc.opt	2005-11-09 07:40:14.006111108 +0100
@@ -2,11 +2,11 @@
 
 TOP=..
 CAMLP4_COMM=TOP=$(TOP) $(TOP)/tools/camlp4_comm.sh pa_ifdef.cmo
-CAMLP4F=-DUNIX
+CAMLP4F=-DUNIX -loc loc
 CAMLP4D=`camlp4 -where`
 LIBUNIX=-cclib -lunix
-OCAMLC=ocamlc.opt -warn-error A
-OCAMLOPT=ocamlopt.opt -warn-error A
+OCAMLC=ocamlc.opt
+OCAMLOPT=ocamlopt.opt
 OCAMLI=
 OCAMLD=
 STRIP=strip
Index: geneweb-4.10/tools/Makefile.inc.out
===================================================================
--- geneweb-4.10.orig/tools/Makefile.inc.out	2005-11-09 07:39:36.643716287 +0100
+++ geneweb-4.10/tools/Makefile.inc.out	2005-11-09 07:40:06.493640285 +0100
@@ -2,11 +2,11 @@
 
 TOP=..
 CAMLP4_COMM=TOP=$(TOP) $(TOP)/tools/camlp4_comm.sh pa_ifdef.cmo
-CAMLP4F=-DUNIX
+CAMLP4F=-DUNIX -loc loc
 CAMLP4D=`camlp4 -where`
 LIBUNIX=-cclib -lunix
-OCAMLC=ocamlc -warn-error A
-OCAMLOPT=ocamlopt -warn-error A
+OCAMLC=ocamlc
+OCAMLOPT=ocamlopt
 OCAMLI=
 OCAMLD=
 STRIP=strip
